### ISSUE TITLE
Works incorrectly with mouse button shortcuts configured in firmware

### ISSUE THREAD

**Issue by nkaretnikov** [pqrs-org/Karabiner-Elements#3013]
i have a built-in mouse config with some shortcuts. specifically, i bind ctrl-up to one of the buttons. with the keyboard, this combination works fine. while using the mouse, it doesn't (when karabiner is installed) -- it just looks like up is being pressed. the even viewer shows the same sequence of presses/actions/keycodes for both the keyboard and mouse. i couldn't figure out what setting affects this, i tried removing all mappings from karabiner. the only solution for me was to uninstall karabiner in the end. i also noticed that the mouse button works a couple of times when you make a change to karabiner configuration (e.g., add/remove a new mapping). maybe there's a timing issue related to how fast the modifier is pressed (when sent from the mouse) or something like that.

**Comment by MuhammedZakir** [comment]
If you don't remap mouse buttons using KE, you may disable input handling for it in KE pereferences. That may fix the issue you're getting.

**Comment by nkaretnikov** [comment]
yep, i disabled the mouse in devices and now it works! ty!

**Comment by MuhammedZakir** [comment]
You're welcome!

When you plugin your mouse, does KE detect it as two devices, or just one? What about `ioreg`[1]?

[1] This command will show (I think): `ioreg -r -d 1 -k LocationID -c AppleUserHIDEventService | less`. Piping to `less` because the output will be very big. Piping to output will allow you to search using vendor/product id.

**Comment by nkaretnikov** [comment]
yes, two devices. here's a screenshot. lmk if you still need the ioreg output

<img width="913" alt="Screen Shot 2022-02-24 at 03 14 24" src="https://user-images.githubusercontent.com/10469885/155432637-a5e22ade-029e-4041-8b2e-ff811026efb0.png">

**Comment by nkaretnikov** [comment]
it's a logitech g502 wireless iirc

**Comment by MuhammedZakir** [comment]
Did you enable input modifications for both? When you mentioned it didn't work, did you mean it correctly send `ctrl+up`, but the shortcut (for `ctrl+up` wasn't triggered, or that it didn't send `ctrl+up` when KE is enabled?
